Do not build OSMesa for a standalone configuration, since Debian only ships
OSMesa in a separate package.

This patch by Thierry Reding.

Index: mesa/src/mesa/Makefile
===================================================================
--- mesa.orig/src/mesa/Makefile	2007-04-25 01:44:31.000000000 +0200
+++ mesa/src/mesa/Makefile	2007-04-25 01:44:58.000000000 +0200
@@ -105,7 +105,7 @@
 	$(OSMESA_DRIVER_OBJECTS)
 
 
-stand-alone: depend subdirs $(TOP)/$(LIB_DIR)/$(GL_LIB_NAME) $(TOP)/$(LIB_DIR)/$(OSMESA_LIB_NAME)
+stand-alone: depend subdirs $(TOP)/$(LIB_DIR)/$(GL_LIB_NAME)
 
 osmesa-only: depend subdirs $(TOP)/$(LIB_DIR)/$(OSMESA_LIB_NAME)
 
